eessi-2023.06-eb-4.7.2-2021a.yml

```diff
@@ -8,3 +8,8 @@ easyconfigs:
   - libjpeg-turbo-2.0.6-GCCcore-10.3.0.eb
   - QuantumESPRESSO-6.7-foss-2021a.eb
   - GROMACS-2021.3-foss-2021a.eb
+  - Qt5-5.15.2-GCCcore-10.3.0.eb:
+# add missing patch files for Qt5 5.15.2 to fix build problems with glibc 2.34,
+# see https://github.com/easybuilders/easybuild-easyconfigs/pull/18087
```

Comment:
@TopRichard Can we match the indent with `options:`, or does that cause trouble?